lib/src/metta/text.rs

```diff
@@ -216,8 +229,8 @@ mod tests {
 
         let mut parser = SExprParser::new("ab");
 
-        assert_eq!(Some(expr!("ab")), parser.parse(&tokenizer));
-        assert_eq!(None, parser.parse(&tokenizer));
+        assert_eq!(Some(expr!("ab")), parser.parse(&tokenizer).unwrap());
+        assert_eq!(None, parser.parse(&tokenizer).unwrap());
```

Comment:
I would use full result as a first argument of `assert_eq!` here and below then if test fails it will show a difference between actual and expected result instead of panic.
```suggestion
        assert_eq!(Ok(Some(expr!("ab"))), parser.parse(&tokenizer));
        assert_eq!(Ok(None), parser.parse(&tokenizer));
```